app/src/main/java/com/example/kotlinfeud/view/fragments/MainFragment.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.kotlinfeud.view.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class MainFragment: BaseFragment () {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return super.onCreateView(inflater, container, savedInstanceState)
+
+        super.onCreateView(inflater, container, savedInstanceState)
+
+        val theView = inflater.inflate(R.layout.)
```

Comment:
these lines are unreachable because you return right after calling the super class.
call the super class, inflate the view, use the view to get variables for the buttons and whatnot, the return theView